Client/Frontend/Browser/FaviconHandler.swift

```diff
@@ -110,18 +51,18 @@ extension FaviconHandler: TabEventHandler {
     tab.favicons.removeAll(keepingCapacity: false)
     Task { @MainActor in
       if let iconURL = metadata.largeIconURL {
-        let (favicon, data) = try await loadFaviconURL(iconURL, type: .appleIcon, forTab: tab)
-        TabEvent.post(.didLoadFavicon(favicon, with: data), for: tab)
+        let favicon = try await loadFaviconURL(iconURL, forTab: tab)
+        TabEvent.post(.didLoadFavicon(favicon, with: nil), for: tab)
```

Comment:
Are there even any places you'd call this with data anymore? If not then we should just remove it from the TabEvent